### PR TITLE
Adding Divider

### PR DESCRIPTION
**Pull request by JacobPrice**

### CODE REVIEW

File: src/components/Divider.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <hr
+    style={{
+      borderTopColor: 'rgba(0, 0, 0, .12)',
+      ...style,
+    }}
+  />
+)
+Divider.propTypes = {
+  color: PropTypes.string,
+  style: PropTypes.object,
```
Comment (tuckerconnelly) [inline src/components/Divider.js]: Separate custom props (color) from regular react props (style) with a line-break--makes the component "interface" more clear.

File: src/components/Divider.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr
+    style={{
+      borderTopColor: 'rgba(0, 0, 0, .12)',
```
Comment (tuckerconnelly) [inline src/components/Divider.js]: Color prop not doing anything :P try `color || 'rgba(0, 0, 0, 0.12)'`

Also might be able to fit whole style prop on one line.

**Review by tuckerconnelly — CHANGES_REQUESTED** [review]

File: src/components/Divider.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      ...style,
+    }}
```
Comment (tuckerconnelly) [inline src/components/Divider.js]: Careful of default browser styles--might wanna go full `<style jsx />` on this bish

File: package.json [hunk 4]
```diff
@@ -54,6 +54,7 @@
     "color": "^1.0.3",
     "lodash": "^4.17.4",
     "prop-types": "^15.5.3",
+    "react": "^15.5.3",
```
Comment (tuckerconnelly) [inline package.json]: Hadn't noticed this--should be a peer dep I think

File: src/components/Divider.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr style={{borderTopColor: color || 'rgba(0, 0, 0, 0.12)', ...style}} />
```
Comment (tuckerconnelly) [inline src/components/Divider.js:5]: Nice! As long as it looks good 😎 

Maybe just remove the `()` for maximum minimalism?

**Review by tuckerconnelly — CHANGES_REQUESTED** [review]

File: src/components/Divider.js (hunk 5 above)
Comment (JacobPrice) [reply src/components/Divider.js:5]: Prettier adds them 😭

**Pull request merged**